docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -375,16 +239,24 @@ Those requirements highlighted in **bold** are included in the NCEPLIBS-external
 
 macOS-specific prerequisites:
 
-* brew install wget
+* brew install gcc@11
 * brew install cmake
+* brew install make
+* brew install wget
 * brew install coreutils
 * brew install pkg-config
-* brew install gnu-sed
+
+..
+   COMMENT: Do we need these last 2? Are there others? (e.g., openssl@3?) Lmod, curl, libtiff
 
```

Comment:
@mkavulich, @willmayfield, can you comment on @gspetro-NOAA's question here?

Reply:
gnu-sed is not needed. Not sure about the others.